src/main/scala/geotrellis/sdg/OutputPyramid.scala

```diff
@@ -29,15 +37,20 @@ object OutputPyramid {
     val pyramid = Pyramid.fromLayerRDD(baseLayer, Some(baseZoom), Some(0))
 
     pyramid.levels.foreach { case (zoom, tileRdd) =>
-      val colorRamp = ColorRamps.Viridis
-      //val Some((min, max)) = histogram.minMaxValues()
-      //val breaks = for { break <- (0 to 64)} yield min + (break * (max-min) / 64)
-      val colorMap = colorRamp.toColorMap(histogram)
+      val Some((min, max)) = histogram.minMaxValues()
+      val width = (max - min) / colorRamp.numStops
+      val breaks = (0 to colorRamp.numStops).map { v => v * width }.toArray
```

Comment:
I'm not sure this matters but you may be off. The default break meaning in ColorMap is "everything less than or equal to this number gets that color". So the number of stops don't match up.

`(0 to colorRamp.numStops)` will generate `numStops + 1` stops, the first one being the value of the min (so potentially it doesn't capture anything?).

This is all "fine" because we get a good picture and that's all this is at the moment but I'm drawing your attention here in case the color breaks have to have specific meaning that is being communicated to us or upstream.